Box.V2/Utility/CrossPlatform.cs

```diff
@@ -34,5 +36,24 @@ public static bool CanConvert<T>(Type objectType)
             return typeof(T).IsAssignableFrom(objectType);
 #endif
         }
+
+        /// <summary>
+        /// Returns the message digest of the stream, formatted as specified by RFC 3230.
+        /// </summary>
+        /// <param name="stream"></param>
+        /// <returns></returns>
```

Comment:
looks like you removed it and make it into public interface, let's talk about this